game/src/core/components/PlayerComponent.java

```diff
@@ -21,6 +23,7 @@
 public final class PlayerComponent implements Component {
 
   private final Map<Integer, InputData> callbacks;
+  private final Map<ILevel, IVoidFunction> onLevelChangeCallbacks = new HashMap<>();
```

Comment:
Das ist für mich Duplicate mit [Game/PreRunConfiguriation onUserLevelLoad](https://github.com/Dungeon-CampusMinden/Dungeon/blob/master/game/src/core/game/PreRunConfiguration.java#L204)

Reply:
moin und erst mal danke fürs review.

wird das denn jedes mal aufgerufen, wenn ein level (also room) gewechselt bzw. erneut geladen wird? ich glaube nicht, insofern wäre es dann kein duplicate.

und diesen pull request wollte ich eigentlich zu machen, weil es in #1497 weitergeht.

Reply:
siehe bitte auch hier https://github.com/Dungeon-CampusMinden/Dungeon/pull/1497/files#diff-52cfefb6ffaff092da4863bf2eca333bbaeea838f17880d9afc7a478c0d4d278 und hier https://github.com/Dungeon-CampusMinden/Dungeon/pull/1497/files#diff-e0489a093d6a66d368ce78dc55bb2500ea3bad1115df0c59afc8dd73bbbd429d

Reply:
> wird das denn jedes mal aufgerufen, wenn ein level (also room) gewechselt bzw. erneut geladen wird? ich glaube nicht, insofern wäre es dann kein duplicate.

Wenn ihr das nicht geändert habt, doch